ocs/ocs_feed.py

```diff
@@ -239,6 +242,7 @@ def publish_message(self, message, timestamp=None):
             # Publish message immediately
             try:
                 self.agent.publish(self.address, (message, self.encoded()))
+                self.message_idx += 1
```

Comment:
Same here; move out of try.

Consider making "increment_message_idx" an optional parameter of `self.encoded()`?